### ISSUE TITLE
Permission denied on fresh install

### ISSUE THREAD

**Issue by ewjoachim** [casperdcl/cli#33]
### Describe the feature or problem you’d like to solve

I have been using gh installed through apt but had to stop because of an [upstream issue](https://github.com/cli/cli/issues/6175). I switched to the snap:

```console
$ sudo snap install gh   
gh 2.6.0-15-g1a10fd5a from Casper (casper-dcl) installed

$ gh repo view --web                    
failed to read configuration:  open /home/me/.config/gh/config.yml: permission denied
```

### Proposed solution

It looks like the snap permissions aren't set correctly.

### Additional context

Ubuntu 20.04

**Comment by lengau** [comment]
Are you still seeing this issue? I was unable to reproduce it myself but I'm not sure whether @casperdcl fixed it.

**Comment by ewjoachim** [comment]
no, they ended up fixing their apt repo, so I switched back to apt

**Comment by Deleplace** [comment]
This error is still occurring in a fresh ubuntu install, after `sudo snap install gh`

**Comment by casperdcl** [comment]
Could you try `sudo snap install --classic gh --channel edge`
